hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/amrmproxy/AMRMProxyService.java

```diff
@@ -111,6 +113,9 @@ public class AMRMProxyService extends CompositeService implements
   private FederationStateStoreFacade federationFacade;
   private boolean federationEnabled = false;
 
+  private final AtomicInteger allocatedCount;
+  private final AtomicInteger askCount;
```

Comment:
Not sure this name is very self explanatory.
Is this the number of requests we got?
requestCount would be more intuitive.

Reply:
Thanks for your help reviewing the code, I will modify the code.